### ISSUE TITLE
Error when deleting world

### ISSUE THREAD

**Issue by GamakCZ** [CzechPMDevs/MultiWorld#15]
- Issue Description: Error when deleting unloaded world

- Version: all

- Server Version: This server is running PocketMine-MP 1.7dev-1183 implementing API version 3.0.0 for Minecraft: PE v1.4.0 (protocol version 261)

- Crashdump, backtrace or other files:
`[17:46:12] [Server thread/INFO]: An unknown error occurred while attempting to perform this command
[17:46:12] [Server thread/CRITICAL]: Unhandled exception executing command 'mw remove test' in multiworld: Call to a member function getFolderName() on null
[17:46:12] [Server thread/CRITICAL]: Error: "Call to a member function getFolderName() on null" (EXCEPTION) in "MultiWorld/src/multiworld/command/subcommand/DeleteSubcommand" at line 50
[17:46:12] [Server thread/DEBUG]: #0 MultiWorld/src/multiworld/command/MultiWorldCommand(110): multiworld\command\subcommand\DeleteSubcommand->executeSub(pocketmine\command\ConsoleCommandSender object, array Array(), string delete)
[17:46:12] [Server thread/DEBUG]: #1 src/pocketmine/command/SimpleCommandMap(258): multiworld\command\MultiWorldCommand->execute(pocketmine\command\ConsoleCommandSender object, string mw, array Array())
[17:46:12] [Server thread/DEBUG]: #2 src/pocketmine/Server(1959): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\command\ConsoleCommandSender object, string mw remove test)
[17:46:12] [Server thread/DEBUG]: #3 src/pocketmine/Server(1944): pocketmine\Server->dispatchCommand(pocketmine\command\ConsoleCommandSender object, string mw remove test)
[17:46:12] [Server thread/DEBUG]: #4 src/pocketmine/Server(1546): pocketmine\Server->checkConsole()
[17:46:12] [Server thread/DEBUG]: #5 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\Server->pocketmine\{closure}()
[17:46:12] [Server thread/DEBUG]: #6 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
[17:46:12] [Server thread/DEBUG]: #7 src/pocketmine/Server(2262): pocketmine\snooze\SleeperHandler->sleepUntil(double 1528998372.8106)
[17:46:12] [Server thread/DEBUG]: #8 src/pocketmine/Server(2133): pocketmine\Server->tickProcessor()
[17:46:12] [Server thread/DEBUG]: #9 src/pocketmine/Server(1714): pocketmine\Server->start()
`

**Comment by MalakasPlayzMC** [comment]
@GamakCZ I am not sure if the issue is valid but I get this error (the 1.4.2 version and API 3.0.0, for MCPE 1.4.0) when I try to load a world:
```
2018-06-14 [19:29:15] [Server thread/ERROR]: Could not load level "Martian": zlib_decode(): data error
2018-06-14 [19:29:15] [Server thread/CRITICAL]: ErrorException: "zlib_decode(): data error" (EXCEPTION) in "vendor/pocketmine/nbt/src/NBTStream" at line 104
2018-06-14 [19:29:15] [Server thread/DEBUG]: #0 (): pocketmine\utils\Utils::errorExceptionHandler(integer 2, string zlib_decode(): data error, string phar:///home/gs/gs/data/servers/7069490/PocketMine-MP.phar/vendor/pocketmine/nbt/src/NBTStream.php, integer 104, array Array())
2018-06-14 [19:29:15] [Server thread/DEBUG]: #1 vendor/pocketmine/nbt/src/NBTStream(104): zlib_decode(string )
2018-06-14 [19:29:15] [Server thread/DEBUG]: #2 src/pocketmine/level/format/io/BaseLevelProvider(51): pocketmine\nbt\NBTStream->readCompressed(string )
2018-06-14 [19:29:15] [Server thread/DEBUG]: #3 src/pocketmine/level/format/io/BaseLevelProvider(45): pocketmine\level\format\io\BaseLevelProvider->loadLevelData()
2018-06-14 [19:29:15] [Server thread/DEBUG]: #4 src/pocketmine/Server(1030): pocketmine\level\format\io\BaseLevelProvider->__construct(string /home/gs/gs/data/servers/7069490/worlds/Martian/)
2018-06-14 [19:29:15] [Server thread/DEBUG]: #5 MultiWorld/src/multiworld/command/subcommand/LoadSubcommand(63): pocketmine\Server->loadLevel(string Martian)
2018-06-14 [19:29:15] [Server thread/DEBUG]: #6 MultiWorld/src/multiworld/command/MultiWorldCommand(110): multiworld\command\subcommand\LoadSubcommand->executeSub(pocketmine\command\ConsoleCommandSender object, array Array(), string load)
2018-06-14 [19:29:15] [Server thread/DEBUG]: #7 src/pocketmine/command/SimpleCommandMap(258): multiworld\command\MultiWorldCommand->execute(pocketmine\command\ConsoleCommandSender object, string mw, array Array())
2018-06-14 [19:29:15] [Server thread/DEBUG]: #8 src/pocketmine/Server(1959): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\command\ConsoleCommandSender object, string mw load Martian)
2018-06-14 [19:29:15] [Server thread/DEBUG]: #9 src/pocketmine/Server(1944): pocketmine\Server->dispatchCommand(pocketmine\command\ConsoleCommandSender object, string mw load Martian)
2018-06-14 [19:29:15] [Server thread/DEBUG]: #10 src/pocketmine/Server(1546): pocketmine\Server->checkConsole()
2018-06-14 [19:29:15] [Server thread/DEBUG]: #11 vendor/pocketmine/snooze/src/SleeperHandler(120): pocketmine\Server->pocketmine\{closure}()
2018-06-14 [19:29:15] [Server thread/DEBUG]: #12 vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
2018-06-14 [19:29:15] [Server thread/DEBUG]: #13 src/pocketmine/Server(2262): pocketmine\snooze\SleeperHandler->sleepUntil(double 1529004555.4373)
2018-06-14 [19:29:15] [Server thread/DEBUG]: #14 src/pocketmine/Server(2133): pocketmine\Server->tickProcessor()
2018-06-14 [19:29:15] [Server thread/DEBUG]: #15 src/pocketmine/Server(1714): pocketmine\Server->start()
2018-06-14 [19:29:15] [Server thread/DEBUG]: #16 src/pocketmine/PocketMine(241): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gs/gs/data/servers/7069490/, string /home/gs/gs/data/servers/7069490/plugins/)
2018-06-14 [19:29:15] [Server thread/DEBUG]: #17 /home/gs/gs/data/servers/7069490/PocketMine-MP.phar(1): require(string phar:///home/gs/gs/data/servers/7069490/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
2018-06-14 [19:29:15] [Server thread/INFO]: [MultiWorld] Level loaded!
```

**Comment by GamakCZ** [comment]
Fixed in 1e78616
